recipes/ibm_db_sa/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "ibm_db_sa" %}
+{% set version = "0.3.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aed72b7275ed5327501f179a0302eaf8934e084aeb3c27a7e0b68f1c83f06173
+
+build:
+  number: 1
+  script: {{ PYTHON }} -m pip install . -vv
+  skip: true  # [linux64]
+
+requirements:
+  build:
+    - sysroot_linux-64 2.17  # [linux64]
+  host:
+    - python
+    - pip
+    - setuptools
+    - libxml2
+
+  run:
+    - python
+    - setuptools
+    - sqlalchemy  1.3.23
```

If it supports everything up to 1.4 (exclusive), it will better allow the full range to make it easier for users to install.
```suggestion
    - sqlalchemy  <1.4
```

---

addressed